composer.json

```diff
@@ -14,20 +14,21 @@
         "issues": "https://github.com/middlewares/cache/issues"
     },
     "require": {
-        "php": "^7.2 || ^8.0",
-        "middlewares/utils": "^3.0",
-        "micheh/psr7-cache": "^0.5.0",
-        "psr/cache": "^1.0",
+        "php": ">=8.1",
+        "middlewares/utils": "^3.3 || ^4.0",
```

Comment:
This must be a major version, so I think it could require only `^4.0` version.

Reply:
No problem - done.

BTW, there's a typo in https://github.com/middlewares/utils/blob/c789be3ce5465928cedc8d1492172ac36918ede4/CHANGELOG.md?plain=1#L15 - it should read "PHP prior to 8.1" not 8.2 :-)

Reply:
hmm, I thought "Prior" means "before", so it removes support for versions before (or prior to) 8.2 (but 8.2 is still supported). Does it makes sense? (I'm not english native, so I'm not sure)

Reply:
Indeed prior means before - but we support 8.1+ here, not 8.2+

Reply:
Oh, you're right. I won't release a new version just for this but I've fixed the changelog.md and edited the release description of GitHub to reflect this change. Thanks!